sdk/storage/azure-storage-queue/src/samples/java/com/azure/storage/queue/QueueJavaDocCodeSamples.java

```diff
@@ -444,8 +444,7 @@ public void clearMessagesWithResponse() {
     }
 
     /**
-     * Code snippet for {@link QueueClient#generateSAS(String, QueueSASPermission, OffsetDateTime, OffsetDateTime,
-     * String, SASProtocol, IPRange)}
+     * Code snippet for {@link QueueClient#generateSas(String, QueueSASPermission, OffsetDateTime, OffsetDateTime, String, SASProtocol, IPRange)}
```

Comment:
Not sure if you own the model class generation, (ie. QueueSASPermission and SASProtocol) and if that is going to be changed.